content/html/concepts/attributes/attributes.md

```diff
@@ -30,29 +30,27 @@ So for example, the `<img>` image element has a mandatory `src` attribute that c
 - `src` is the attribute name.
 - `codecademy_logo.png` is the attribute value.
 
-All HTML elements can have attributes. Multiple of them, too!
+All HTML elements can have attributes; many have more than one.
 
 ```html
 <img src="nyan_cat.gif" width="500" height="600" />
 ```
 
-**Note:** Attribute names are case insensitive, but it is a good practice to keep HTML markup lowercase.
+**Note:** Attribute names are not case-sensitive, but it is good practise to write HTML markup in lowercase.
 
 ## Unique ID Attributes
 
-In HTML, specific and unique `id` attributes can be assigned to different elements in order to differentiate between them.
+Specific and unique `id` attributes can be assigned to elements to differentiate them, and `id` values can be referenced in CSS and JavaScript to manipulate, format, and perform specific instructions on a single element alone, without affecting others of the same type.
 
-When needed, the `id` value can be called upon by CSS and JavaScript to manipulate, format, and perform specific instructions on that element and that element only.
-
-Valid `id` attributes should begin with a letter and should only contain letters (`a`-`Z`), digits (`0`-`9`), hyphens (`-`), underscores (`_`), and periods (`.`).
+Valid `id` attribute values begin with a letter and must comprise only letters (`a`-`Z`), digits (`0`-`9`), hyphens (`-`), underscores (`_`), and periods (`.`).
```

Comment:
```suggestion
Valid `id` attribute values begin with a letter and must be comprised of only letters (`a`-`Z`), digits (`0`-`9`), hyphens (`-`), underscores (`_`), and periods (`.`).
```

Reply:
Sorry, I don't agree that "comprised of" is grammatically correct. "Must comprise" reads fine to me.

Reply:
Whoops, I meant to write "composed of" as per https://www.merriam-webster.com/words-at-play/can-you-use-comprised-of-grammar. Thanks for pointing out my error here 👍 We'll just keep this as "must comprise only." Go ahead and resolve this convo